contrib/qat/compression/qatzip/compressor/test/qatzip_compressor_impl_test.cc

```diff
@@ -49,7 +49,21 @@ class QatzipCompressorImplTest : public ::testing::Test {
     ASSERT_EQ(original_text.length(), decompressed_text.length());
     EXPECT_EQ(original_text, decompressed_text);
   }
-  uint32_t default_input_size_{796};
+
+  Envoy::Compression::Compressor::CompressorFactoryPtr
+  createQatzipCompressorFactoryFromConfig(const std::string& json) {
+    envoy::extensions::compression::qatzip::compressor::v3alpha::Qatzip qatzip_config;
+    TestUtility::loadFromJson(json, qatzip_config);
+
+    return qatzip_compressor_library_factory_.createCompressorFactoryFromProto(qatzip_config,
+                                                                               context_);
+  }
+  // A value which has an impact on the size of random input created for the tests that use
```

Comment:
Hmm, can you explain this a bit more? Is this an arbitrary number?

Reply:
https://github.com/intel/envoy/blob/7ff7e5770630394095d0135c652acff20ef07b2b/contrib/qat/compression/qatzip/compressor/test/qatzip_compressor_impl_test.cc#L26-L30

The verifyWithDecompressor function (also used in a very similar way in other compression unit tests) creates random input to compress here. `default_input_size_ * i` is the size of the random input to be compressed during one iteration of the for loop. The actual number could be changed since it is just a value to be used for generating input, but the same number is used in other unit tests as well, for example:

https://github.com/intel/envoy/blob/7ff7e5770630394095d0135c652acff20ef07b2b/test/extensions/compression/brotli/compressor/brotli_compressor_impl_test.cc#L54

https://github.com/intel/envoy/blob/7ff7e5770630394095d0135c652acff20ef07b2b/test/extensions/compression/zstd/compressor/zstd_compressor_impl_test.cc#L56

Reply:
Ok I get it. If this number is used for all tests then it's good.